openmetadata-ui/src/main/resources/ui/cypress/common/common.js

```diff
@@ -13,9 +13,12 @@
 
 /// <reference types="cypress" />
 
+import { SEARCH_INDEX } from '../constants/constants';
+
 export const descriptionBox =
   '.toastui-editor-md-container > .toastui-editor > .ProseMirror';
 export const uuid = () => Cypress._.random(0, 1e6);
+const RETRY_TIMES = 30;
```

Comment:
With the retry count of 30 and a waiting time of 20 seconds between each try, we are waiting a whole 10 minutes for any ingestion job to succeed. We should not be retrying these many times in my opinion.

Reply:
Hi @vivekratnavel, We are not waiting for a total of 10 min for every spec, as ingestion status update is entirely dependent on airflow ingestion we are not sure about when it is going to be successful, so we had to retry until it's successful but we are just providing an upper limit to it! also, it's old logic I have just moved the variable to an outside function so we can utilize this if needed!